test/lit/wasm-split/instrument-in-secondary-memory-custom-names.wast

```diff
@@ -0,0 +1,92 @@
+;; RUN: wasm-split %s --instrument --in-secondary-memory --import-namespace=custom_env --secondary-memory-name=custom_name -all -S -o - | filecheck %s
+
+;; Check that the output round trips and validates as well
+;; RUN: wasm-split %s --instrument --in-secondary-memory -all -g -o %t.wasm
+;; RUN: wasm-opt -all %t.wasm -S -o -
+
+(module
+  (import "env" "foo" (func $foo))
+  (export "bar" (func $bar))
+  (memory $0 1 1)
+  (func $bar
+    (call $foo)
+  )
+  (func $baz (param i32) (result i32)
+    (local.get 0)
+  )
+)
+
+;; Check that a memory import has been added for secondary memory
+;; CHECK: (import "custom_env" "custom_name" (memory $custom_name (shared 1 1)))
+
+;; And the profiling function exported
```

Comment:
Everything from here and below is the same as in the other test. Since we update these tests by hand, it's probably best to minimize the toil and remove the extra checks here.

Reply:
Just want to call out that the "custom_name" memory name is used in three spots below on lines 31, 39, and 76.

Reply:
I deleted everything that was duplicated in instrument-in-secondary-memory.wast and kept three lines prepended with a "CHECK:" to handle 31, 39, and 76.